src/main/scala/org/broadinstitute/hail/methods/Pedigree.scala

```diff
@@ -86,6 +87,27 @@ object Pedigree {
   // plink only prints # of kids under CHLD, but the list of kids may be useful, currently not used anywhere else
   def nuclearFams(completeTrios: Array[CompleteTrio]): Map[(String, String), Array[String]] =
     completeTrios.groupBy(t => (t.dad, t.mom)).mapValues(_.map(_.kid)).force
+
+  def gen(sampleIds: IndexedSeq[String]): Gen[Pedigree] = {
+    Gen.shuffle(sampleIds)
+      .map { is =>
+        val groups = is.grouped(3)
+        val trios = groups.map { g =>
+          val (kid, mom, dad) = (g(0),
+            if (g.length >= 2) Some(g(1)) else None,
+            if (g.length >= 3) Some(g(2)) else None)
+          Trio(kid, fam = None, mom = mom, dad = dad, sex = None, pheno = None)
```

Comment:
Make this more random: don't include all samples in the pedigree (50%?) and make some parents missing (20%?)